src/daemon.js

```diff
@@ -121,7 +122,12 @@ class Daemon {
    */
   _cleanUnixSocket () {
     return new Promise((resolve, reject) => {
-      rimraf(this._sock, (err) => {
+      const path = this._addr.toString().split('/unix')
```

Comment:
`this._addr.getPath()` should return you the file path. If there's no path it will return null. It was added with the unix addition to `js-multiaddr`.